icubam/__init__.py

```diff
@@ -0,0 +1,14 @@
+import pkg_resources
+from os import path
+from setuptools.config import read_configuration
+
+def _extract_version(package_name):
```

Comment:
Ideally __init__ should not import setuptools or load any files. 

The standard way is rather to indicate `__version__` here and load it from setup.py.

Reply:
This is what I usually do, but I thought it was better to put it in the `setup.cfg`?

Reply:
You can indeed put it in `setup.cfg`, but the version in `__init__.py` should be self contained and not depend on any other file. You are not guaranteed that `setup.cfg` will be there once you pip install the package. Also performance wise it's bad. Which could mean two files that need editing on version update.

https://github.com/pypa/setuptools_scm might be useful to automate versioning somewhat BTW.